lms/envs/common.py

```diff
@@ -1755,6 +1755,7 @@
     'js/student_profile/views/learner_profile_factory': 'js/student_profile/views/learner_profile_factory.js',
     'js/courseware/courseware_factory': 'js/courseware/courseware_factory.js',
     'js/groups/views/cohorts_dashboard_factory': 'js/groups/views/cohorts_dashboard_factory.js',
+    'js/groups/views/discussions_dashboard_factory': 'js/groups/views/discussions_dashboard_factory.js',
```

Comment:
I've read the comment before this block several times, but still don't understand this mapping. It looks like this is something Toby added in February-- I will have to read up on the details unless you can explain it to me (in person). :)